changelog/7978.md

```diff
@@ -0,0 +1,3 @@
+### Changes in the README
+
+I added to the readme some notes and new comands that helped me with the installation of the a+ repository locally.  
```

Comment:
we don't use personal pronouns in the changelog, it needs to be neutral writing.
we are following this format :  https://keepachangelog.com/en/1.1.0/
Titles can be:

>
    Added for new features.
    Changed for changes in existing functionality.
    Deprecated for soon-to-be removed features.
    Removed for now removed features.
    Fixed for any bug fixes.
    Security in case of vulnerabilities.

So the changelog would be more like the following:
```
### Changed 

-  Update README with some notes and new comands for the installation of the a+ repository locally. 
```